data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/elasticsearch.mapping.yml

```diff
@@ -0,0 +1,6 @@
+pluginName: opensearch
+mappedAttributes:
```

Comment:
I also believe that this needs:

```
cacert: cert
ssl: insecure
proxy: proxy
```

Reply:
I will add these once I get the final mapping names from Han.